cfgmgr/vlanmgr.cpp

```diff
@@ -147,19 +147,26 @@ bool VlanMgr::removeHostVlanMember(int vlan_id, const string &port_alias)
 {
     stringstream cmd;
     string res;
+    int ret;
 
     cmd << BRIDGE_CMD << " vlan del vid " << vlan_id << " dev " << port_alias;
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
 
     cmd.str("");
     // When port is not member of any VLAN, it shall be detached from Dot1Q bridge!
     cmd << BRIDGE_CMD << " vlan show dev " << port_alias << " | " << GREP_CMD << " None";
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
-    if (!res.empty())
-    {
-        cmd.str("");
-        cmd << IP_CMD << " link set " << port_alias << " nomaster";
-        EXEC_WITH_ERROR_THROW(cmd.str(), res);
+    ret = swss::exec(cmd.str(), res);
+    ret = WEXITSTATUS(ret);
```

Comment:
@JipanYanga , It looks like this WEXITSTATUS is better to be hided in the swss::exec(). 

Inside swss::exec(). The pclose() function waits for the associated process to terminate and returns the exit status of the command as returned by wait4(2).

pclose() returns -1 if wait4(2) returns an error.

Reply:
@lguohan putting `WEXITSTATUS` in `swss:exec()` doesn't break anything? `WEXITSTATUS` loses some information about the child process (e.g. received signal ). If this is not the case, I will put it in `swss:exec()`.

Reply:
@ishidawataru @lguohan For shell command execution in current implementation,  for most of the cases  the return value of zero is expected, signal interruption should trigger exception.   But for this case and https://github.com/Azure/sonic-swss/blob/master/cfgmgr/vlanmgr.cpp#L38 ,  we know the return value may be non-zero.    It looks to me  user of swss:exec() should have the original returned value and process it accordingly, though sometimes it is safe to ignore the return value.   If we don't put  WEXITSTATUS in swss:exec() ,  probably more comments including how to use WEXITSTATUS could be added in the header file?

Reply:
@JipanYanga Thanks for your comment. Yes, I also think keeping returning value of `swss::exec()` as is and adding comments would be better.

@lguohan WDYT?